service/api/tpl/wp-config.tpl

```diff
@@ -154,6 +154,7 @@ if ( true === WP_CACHE ) {
  * @link https://codex.wordpress.org/Debugging_in_WordPress
  */
 define( 'WP_DEBUG', ! $is_gae );
+define( 'WP_DEBUG_LOG', true );
```

Comment:
This would also need to only be set for local not on App Engine. So instead of `true` it would be `! $is_gae`.

Reply:
The `debug.log` file will only be created when WP_DEBUG is true so it will not be created in production.

I can change it to the variable.